HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -2184,6 +2184,81 @@ def test_air_source_heat_pumps
+    hpxml_bldg.heat_pumps[0].heating_capacity_retention_temp = nil
+    XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+    _default_hpxml, default_hpxml_bldg = _test_measure()
+    # Heating
+    max_dp_47f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 47.0 && dp.capacity_description == HPXML::CapacityDescriptionMaximum }
+    nom_dp_47f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 47.0 && dp.capacity_description == HPXML::CapacityDescriptionNominal }
+    min_dp_47f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 47.0 && dp.capacity_description == HPXML::CapacityDescriptionMinimum }
+    max_dp_17f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 17.0 && dp.capacity_description == HPXML::CapacityDescriptionMaximum }
+    nom_dp_17f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 17.0 && dp.capacity_description == HPXML::CapacityDescriptionNominal }
+    min_dp_17f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 17.0 && dp.capacity_description == HPXML::CapacityDescriptionMinimum }
+    max_dp_5f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 5.0 && dp.capacity_description == HPXML::CapacityDescriptionMaximum }
+    nom_dp_5f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 5.0 && dp.capacity_description == HPXML::CapacityDescriptionNominal }
+    min_dp_5f = default_hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == 5.0 && dp.capacity_description == HPXML::CapacityDescriptionMinimum }
+    # 47F
+    assert_in_epsilon(9576.6, max_dp_47f.capacity, 0.01)
+    assert_in_epsilon(8700.0, nom_dp_47f.capacity, 0.01)
+    assert_in_epsilon(2601.3, min_dp_47f.capacity, 0.01)
+    assert_in_epsilon(2.34, max_dp_47f.efficiency_cop, 0.01)
+    assert_in_epsilon(2.49, nom_dp_47f.efficiency_cop, 0.01)
+    assert_in_epsilon(3.20, min_dp_47f.efficiency_cop, 0.01)
+    # 17F
+    assert_in_epsilon(9176.9, max_dp_17f.capacity, 0.01)
+    assert_in_epsilon(7500.0, nom_dp_17f.capacity, 0.01)
+    assert_in_epsilon(3127.5, min_dp_17f.capacity, 0.01)
+    assert_in_epsilon(1.66, max_dp_17f.efficiency_cop, 0.01)
+    assert_in_epsilon(1.84, nom_dp_17f.efficiency_cop, 0.01)
+    assert_in_epsilon(2.08, min_dp_17f.efficiency_cop, 0.01)
+    # 5F
+    assert_in_epsilon(7950.8, max_dp_5f.capacity, 0.01)
+    assert_in_epsilon(7857.4, nom_dp_5f.capacity, 0.01)
+    assert_in_epsilon(2552.7, min_dp_5f.capacity, 0.01)
+    # The RESNET table seems to be incorrect in calculating the Net Power @ 5F,
```

Comment:
@shorowit @jonwinkler Probably we should let Neal know this error? It's the calculation in spreadsheet.

Reply:
Yueyue sent an email to Neal, who fixed the bug.